ReText/window.py

```diff
@@ -217,6 +217,8 @@ def __init__(self, parent=None):
 			lambda: self.searchBar.setVisible(False),
 			shct=QKeySequence.Cancel)
 		self.actionCloseSearch.setPriority(QAction.LowPriority)
+		self.actionInsertImage = self.act(self.tr('Image'), 'image',
+										  self.insertImage)
```

Comment:
Please use tabs for indentation but spaces for alignment, so

```suggestion
		                                  self.insertImage)
```